app/views/propertyrepresentation/revokeAgentProperties.scala.html

```diff
@@ -48,14 +54,11 @@ <h1 class="heading-large">@Messages("propertyRepresentation.removeAgentPropertie
         <a href="#" id="par-select-all-top">@Messages("propertyRepresentation.appointAgentProperties.selectDeselect")</a>
     </div>
 
-    <form method="GET" action="@agentAppointment.routes.AppointAgentController.selectAgentPropertiesSearchSort(pagination)">
-        <input type="hidden" name="agentCode" value="@pagination.agentCode">
-        <input type="hidden" name="checkPermission" value="@pagination.checkPermission.name">
-        <input type="hidden" name="challengePermission" value="@pagination.challengePermission.name">
-        <input type="hidden" name="pageNumber" value="@pagination.pageNumber">
+    <form method="GET" action="@agentAppointment.routes.AppointAgentController.selectAgentPropertiesSearchSort(pagination, params, agentCode)">
+        <input type="hidden" name="pageNumber" value="@pagination.page">
         <input type="hidden" name="pageSize" value="@pagination.pageSize">
-        <input type="hidden" name="sortField" value="@pagination.sortField">
-        <input type="hidden" name="sortOrder" value="@pagination.sortOrder">
+        <input type="hidden" name="sortField" value="@params.sortfield">
+        <input type="hidden" name="sortOrder" value="@params.sortorder">
```

Comment:
Just verify once again Property name camelCase
 sortField vs sortfield,  sortOrder vs sortorder 

Last week, We had 2 bugs around this camelCase vs non camel case